engine/schema/src/main/resources/META-INF/db/schema-41500to41510.sql

```diff
@@ -18,4 +18,85 @@
+INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) SELECT UUID(),'VMware', '7.0', guest_os_name, guest_os_id, utc_timestamp(), 0  FROM `cloud`.`guest_os_hypervisor` WHERE hypervisor_type='VMware' AND hypervisor_version='6.7';
+
+-- Add support for crxPod1Guest from VMware 7.0
+INSERT INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (330, UUID(), 7, 'crxPod1Guest', now());
+INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '7.0', 'darwin19_64Guest', 330, now(), 0);
+
+-- Add support for darwin19_64Guest from VMware 7.0
+INSERT INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (331, UUID(), 7, 'macOS 10.15 (64 bit)', now());
+INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '7.0', 'darwin19_64Guest', 331, now(), 0);
+
+-- Add support for debian11_64Guest from VMware 7.0
+INSERT INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (332, UUID(), 2, 'Debian GNU/Linux 11 (64-bit)', now());
+INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '7.0', 'debian11_64Guest', 332, now(), 0);
+
+-- Add support for debian11Guest from VMware 7.0
+INSERT INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (333, UUID(), 2, 'Debian GNU/Linux 11 (32-bit)', now());
+INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '7.0', 'debian11Guest', 333, now(), 0);
+
+-- Add support for vmkernel7Guest from VMware 7.0
+INSERT INTO `cloud`.`guest_os` (id, uuid, category_id, display_name, created) VALUES (334, UUID(), 7, 'vmkernel7Guest', now());
+INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '7.0', 'vmkernel7Guest', 334, now(), 0);
+
+-- Add support for windows2019srv_64Guest from VMware 7.0
+INSERT INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervisor_version, guest_os_name, guest_os_id, created, is_user_defined) VALUES (UUID(),'VMware', '7.0', 'windows2019srv_64Guest', 276, now(), 0);
+
+
+-- Add support for VMware 7.0.1.0
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities` (uuid, hypervisor_type, hypervisor_version, max_guests_limit, security_group_enabled, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported, vm_snapshot_enabled) values (UUID(), 'VMware', '7.0.1.0', 1024, 0, 59, 64, 1, 1);
```

Comment:
Should the version be just `7.0.1`? 
I see this on esxi - `/resourceGroups/version = "7.0.1"`

Reply:
or is it the version returned by vcenter?

Reply:
I've checked the code and this value comes from the SDK interaction to the hostMO.getHostAboutInfo().getApiVersion();

In a test environment I get:
````
MariaDB [cloud]> select id,name,status,version, hypervisor_version, hypervisor_type from host where type = 'Routing';
+----+-------------+--------+-------------------+--------------------+-----------------+
| id | name        | status | version           | hypervisor_version | hypervisor_type |
+----+-------------+--------+-------------------+--------------------+-----------------+
|  1 | 10.0.33.193 | Up     | 4.15.1.0-SNAPSHOT | 7.0.1.0            | VMware          |
|  2 | 10.0.35.144 | Up     | 4.15.1.0-SNAPSHOT | 7.0.1.0            | VMware          |
+----+-------------+--------+-------------------+--------------------+-----------------+
2 rows in set (0.00 sec)

````